indico/modules/events/abstracts/templates/abstract/review.html

```diff
@@ -18,13 +19,54 @@
                     {%- endtrans %}
                 </h3>
                 {% for review in reviews %}
-                    {{ _render_review(review) }}
+                    {{ _render_review(abstract.event_new.cfa, review) }}
                 {% endfor %}
             {% endif %}
         {% endfor %}
     </div>
 {% endmacro %}
 
+{% macro render_review_boxes(abstract, review_forms) %}
+    {% for track in abstract.reviewed_for_tracks|sort(attribute='title')
+       if track.is_user_reviewer(session.user) %}
+        <div id="abstract-review-box-{{ track.id }}" class="i-timeline-item">
+            {{ render_review_box(review_forms[track.id], abstract, track) }}
+        </div>
+    {% endfor %}
+{% endmacro %}
+
+{% macro render_review_box(form, abstract, track) %}
+    <div class="i-timeline-item-label action icon-checkmark"></div>
+    <div class="i-timeline-item-box content-indicator">
+        <div class="i-box-content">
+            <div class="flex">
+                <div class="stretch">
+                    <strong>
+                        {% trans track_title=track.title -%}
+                            Review for track "{{ track_title }}"
+                        {%- endtrans %}
+                    </strong>
+                </div>
+            </div>
+        </div>
+        <div class="i-box-footer">
+            {{ form_header(form, orientation='vertical', id=('track-review-form-' + track.id|string)) }}
+            {{ form_rows(form, skip=('comment', 'proposed_action')) }}
+            {{ form_rows(form, fields=('comment', 'proposed_action')) }}
+            {% set form_id = '#track-review-form-' + track.id|string %}
+            {% call form_footer(form, skip_label=skip_labels) %}
+                <button class="i-button big highlight"
+                        data-update="#abstract-review-box-{{ track.id }}"
+                        data-params-selector='{{ form_id }} select, {{ form_id }} input[type="radio"]:checked, {{ form_id }} input[type="hidden"], {{ form_id }} textarea'
+                        data-method="POST"
+                        data-href="{{ url_for('.review_abstract', abstract, track) }}">
+                    {% trans %}Review{% endtrans%}
```

Comment:
To be exact, the reviewer has already reviewed the abstract at this point. I would maybe rename it to `Send review` or `Submit review`.